azure-communication-ui/azure-communication-ui/src/main/java/com/azure/android/communication/ui/presentation/fragment/setup/SetupViewModel.kt

```diff
@@ -67,7 +67,8 @@ internal class SetupViewModel(
             state.permissionState,
             state.localParticipantState.cameraState,
             state.localParticipantState.audioState,
-            state.callState.CallingStatus,
+            state.callState,
+            this::openAudioDeviceSelectionMenu
```

Comment:
is there anyone else using `SetupViewModel.openAudioDeviceSelectionMenu()`?  If not can we replace with direct call:  `audioDeviceListViewModel::displayAudioDeviceSelectionMenu`

Reply:
Done